torchaudio/transforms.py

```diff
@@ -604,6 +604,13 @@ class MuLawEncoding(torch.nn.Module):
 
     Args:
         quantization_channels (int, optional): Number of channels. (Default: ``256``)
+
+    Example
+
+       >>> waveform, sample_rate = torchaudio.load('test.wav', normalization=True)
```

Comment:
nit: ``normalization`` is changed to ``normalize`` now. Let's replace it. 
```suggestion
       >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
```